concent_api/concent_api/settings/base.py

```diff
@@ -369,4 +369,4 @@
 SCI_CALLBACK_RETRIES_TIME = 60
 
 # Define chain where PaymentInterface has to connect to geth
-# PAYMENT_INTERFACE_CHAIN = golem_sci.chains.RINKEBY
+# PAYMENT_INTERFACE_CHAIN =
```

Comment:
I think that `PAYMENT_INTERFACE_CHAIN` will sound confusing for someone who is only configuring Concent and does not even know what "payment interface" is or does. Wouldn't `ETHEREUM_CHAIN` be a better name?

Reply:
Alright, name changed.

Reply:
I will change commit name during sqashing